apps/src/music/redux/musicRedux.ts

```diff
@@ -49,6 +49,9 @@ export interface MusicState {
   hideHeaders: boolean;
   /** The current list of playback events */
   playbackEvents: PlaybackEvent[];
+  /** The playback events associated with the exemplar player */
+  exemplarPlaybackEvents: PlaybackEvent[];
+  playerContext: 'exemplar' | 'musicView' | null;
```

Comment:
I'm thinking we may not need this state; we already have an `isPlaying` boolean that tracks if the main lab player is playing. The exemplar player can listen to that and stop itself whenever `state.music.isPlaying` is true. And then it already has the ability to stop the main player when it wants to start playing - so that may be sufficient?

Reply:
Thanks, I didn't realize we already had the boolean! Updated: https://github.com/code-dot-org/code-dot-org/pull/64114/commits/03c6d37ba908a81373cb38e615638ded1f79e47c

Reply:
I ended up fixing this as well as your other point about `labSetPlaying` over two commits:

https://github.com/code-dot-org/code-dot-org/pull/64114/files/8a5b9c659ac1f6abbf828fadb306673870dcaf4d..2eba2148b8b7e222869c18d5abeba54220560084